infrastrucutre/subnets.tf

```diff
@@ -0,0 +1,47 @@
+module "subnet_addrs" {
```

Comment:
Combine this file with the VPC module in vpc.tf or network.tf, just to make relevant to each other stay in the same file.